dime-research-standards/pillar-1-research-ethics/research-ethics-resources/informed-consent-templates/informed-consent-template-written-covid.md

```diff
@@ -0,0 +1,197 @@
+•	maintaining a distance of 2m/6ft
+•	wearing masks and gloves throughout
+•	providing you with a mask
+•	remaining outside  
+•	<any other protocols>
+We are also willing to follow any extra measure that would make you feel more comfortable when taking the survey.
+If in the next couple of weeks, we realize that we might have put you at risk (discovering for instance that a member of our team contracted the COVID-19 or has been in contact with a sick person), we will immediately contact you and refer you to the closest health facility to get tested.
+
+
+## Potential benefits
+
+**[Describe benefits to subjects expected from the research.
+If the subject will not benefit from participation, clearly state this fact.]**
+
+**[State the potential benefits, if any, to science or society expected from the research.]**
+
+## Payment for participation
+
+**[State whether the subject will receive payment. If not, state so.
+If the subject will receive payment, describe remuneration amount, when payment is scheduled,
+and proration schedule should the subject decide to withdraw or is withdrawn by the investigator.]**
+
+Legally, you can be paid only if you are a US citizen, a legal resident alien (e.g., possess a “green” card),
+or have a work eligible visa sponsored by the paying institution.
+You will also be reimbursed for transportation costs to and from the experiment site.
+
+## Privacy and confidentiality
+
+Any information obtained in connection with this study and that can be identified with you will remain confidential and will
+be disclosed only with your permission or as required by law.
+Information shared by you might be published without any identifiable information being shared as the research requires this.
+Note that complete confidentiality of the responses you provide will be ensured when publishing the data.
+
+The **[government/ministry name]** might have access to all information you provide as they are funding this study.
+
+**[If information will be released to any other party for any reason,
+state the person/agency to whom the information will be furnished, the nature of the information,
+and the purpose of the disclosure.]**
+
+**[If activities are to be audio- or videotaped, specific consent for this must be obtained.
+Also describe the subject's right to review/edit the tapes, who will have access to them,
+if they will be used for educational purpose, and when they will be erased.]**
+
+- **[Give a brief description of how personal information, research data, and related records will be coded, stored, etc.
+to prevent access by unauthorized personnel.]**
+- **[State if and when individual data will be destroyed following analysis, and the format in which data will be reported.]**
+- **[Explain how specific consent will be solicited, if any other uses are contemplated.]**
+
+## Identification of investigators
```

Comment:
why in this case is the contact for the investigators and not the survey firm? it seems preferable to list survey firm especially since they are more likely to have local contact